server/classes/sockets/RoomSocketBinder.ts

```diff
@@ -0,0 +1,46 @@
+import { Socket } from 'socket.io';
+import PlayerFactory from '../../factories/PlayerFactory';
+import Application from "../Application";
+import SocketBinder from "./SocketBinder";
+
+export default class RoomSocketBinder extends SocketBinder {
+    static bindSocket(socket: Socket): void {
+        this.onJoinRoom(socket);
+        this.onMessageRoom(socket);
+        this.onDrawing(socket);
+    }
+
+    private static onJoinRoom(socket: Socket): void {        
+        socket.on("join-room", (roomId, ack) => {
+            if(Application.getRoomStorage().exists(roomId)) {
+                socket.join(`room-${roomId}`)
+                const updatedRoom = Application.getRoomStorage().addPlayer(roomId, PlayerFactory.createPlayer(socket));
+                socket.to(`room-${roomId}`).emit("update-room", updatedRoom);
```

Comment:
We could separate `room-${roomId}` string into a static method in the Room.ts class that return the room's name.
Like `socket.to(Room.getRoomName(roomId)).emit("update-room", updatedRoom);` where `getRoomName(roomId)` returns `"room-<roomId>"`.